src/main/resources/config/application-dev.yml

```diff
@@ -1,8 +1,8 @@
 spring:
   data:
     neo4j:
-      # Automatically create and update indexes and constraints.
-      auto-index: update
+      # Nothing is done with index and constraint annotations.
+      auto-index: none
```

Comment:
We can simply remove the auto-index config here because `spring.data.neo4j.auto-index=none` is the default anyway.